private/smithy-rpcv2-cbor/test/functional/rpcv2cbor.spec.ts

```diff
@@ -235,18 +235,21 @@ it("empty_input:Request", async () => {
     const r = err.request;
     expect(r.method).toBe("POST");
     expect(r.path).toBe("/service/RpcV2Protocol/operation/EmptyInputOutput");
-    expect(r.headers["content-length"]).toBeDefined();
+    expect(
+      r.headers["content-length"],
+      `Header key "content-length" should have been defined in ${JSON.stringify(r.headers)}`
+    ).toBeDefined();
 
-    expect(r.headers["x-amz-target"]).toBeUndefined();
+    expect(
+      r.headers["x-amz-target"],
+      `Header key "x-amz-target" should have been undefined in ${JSON.stringify(r.headers)}`
+    ).toBeUndefined();
 
-    expect(r.headers["accept"]).toBeDefined();
```

Comment:
an `isDefined` assertion is unnecessary when preceding a match to an exact value.